install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -309,3 +309,133 @@ $ ansible-playbook [-i /path/to/file] \
 
 . After the playbook completes successfully,
 xref:../install_config/install/advanced_install.adoc#advanced-verifying-the-installation[verify the installation].
+
+[[replacing-existing-masters]]
+== Replacing Existing Masters
+
+. Back up etcd from the primary
+xref:../admin_guide/backup_restore.adoc#etcd-backup[etcd] and
+xref:../admin_guide/backup_restore.adoc#master-backup[master].
++
+[IMPORTANT]
+====
+Ensure that you back up the *_/etc/etcd/ca/_* directory, as noted in the
+xref:../admin_guide/backup_restore.adoc#backup-restore-adding-etcd-hosts[Backup
+and Restore] topic.
+====
+
+. Provision as many new virtual machines (VMs) as there are masters to replace.
+
+. Add or expand the cluster. If you want to add 3 masters with etcd collated,
+ repeat this section 3 times.
+
+.. Add a master using the xref:adding-nodes-advanced[master *_scaleup.yml_*
+playbook] on the new data center. In step 3 of that process, add the host of the
+new data center in `[new_masters]` and `[new_nodes]`.
+
+.. Put the same host in the xref:adding-etcd-hosts-to-existing-cluster[etcd]
+section and run the etcd *_scaleup.yml_* playbook.
+
+.. Verify that the host was added:
++
+----
+# oc get nodes
+----
+
+.. Verify that the master host IP was added:
++
+----
+# oc get ep kubernetes
+----
+
+.. Verify that etcd was added:
++
+----
+# ETCDCTL_API=2
+# source /etc/etcd/etcd.conf
+# etcdctl --cert-file=$ETCD_PEER_CERT_FILE --key-file=$ETCD_PEER_KEY_FILE \
+  --ca-file=/etc/etcd/ca.crt --endpoints=$ETCD_LISTEN_CLIENT_URLS member list
+----
+
+. Stop old masters.
+
+.. Shut down the services:
++
+----
+# systemctl stop atomic-openshift-master-api
+----
+
+.. Copy *_/etc/origin/master/ca.crt_*, *_/etc/origin/master/ca.key_*, and
+*_/etc/origin/master/ca.serial.txt_* from the *_/etc/origin/master_* directory
+to the new master host that is listed first in your hosts file. This directory
+is on the old master host that was listed first.
```

Comment:
In my test, we only need to copy `/etc/origin/master/ca.serial.txt`. The rest of the files are exactly the same.